e2e-tests/src/observe/get_posix_signals_stream.rs

```diff
@@ -0,0 +1,102 @@
+#[cfg(test)]
+mod tests {
+    use std::{sync::Arc, time::Duration};
```

Comment:
e2e tests are cool, but do we even have unit tests for this?

Reply:
See the discussion here: https://github.com/aurae-runtime/aurae/issues/365

We have a unit test in the `observe` service, but once we are going to test observing specific workloads we're going to need to do a lot of gymnastics on that unit test.